bindings/c/src/lib.rs

```diff
@@ -46,6 +46,29 @@ pub unsafe extern "C" fn libsql_sync(
     }
 }
 
+#[no_mangle]
+pub unsafe extern "C" fn libsql_sync2(
+    db: libsql_database_t,
+    out_replicated: *mut replicated,
+    out_err_msg: *mut *const std::ffi::c_char,
+) -> std::ffi::c_int {
+    let db = db.get_ref();
+    match RT.block_on(db.sync()) {
+        Ok(replicated) => {
+            if out_replicated != std::ptr::null_mut() {
```

Comment:
Is this different than `!out_replicated.is_null()` ?

Reply:
oh no its not I forget that you can call that on a ptr, let me try that